.NET/Microsoft.Recognizers.Text.DateTime/German/Parsers/GermanTimeParserConfiguration.cs

```diff
@@ -105,13 +105,13 @@ public void AdjustByPrefix(string prefix, ref int hour, ref int min, ref bool ha
                     minStr = match.Groups["deltaminnum"].Value;
                     deltaMin = Numbers[minStr];
                 }
-            }
 
-            // @TODO move hardcoded values to resources file
+                // @TODO move hardcoded values to resources file
 
-            if (trimmedPrefix.EndsWith("zum", StringComparison.Ordinal))
```

Comment:
What happens to the cases with "zum"?

Reply:
Or was it just a case of incorrect localization?

Reply:
@tellarin To my knowledge, the term "zum" is not used in German to indicate time. The preposition "vor" is most commonly used to indicate a specific time or duration prior to a certain event or moment. Please refer to the following articles for more information:
* [Time Expressions in German](http://www.dartmouth.edu/~deutsch/Grammatik/Zeit/Time.html)
* [German Time Phrases & Expressions](https://zinglanguages.com/knowledgehub/german-time-phrases-expressions/)

Also I have not found any tests for the term "zum" in relation to German time.
In any case, this code does not do anything because the TimePrefix regex does not contain "zum":
https://github.com/microsoft/Recognizers-Text/blob/b80b58ce49d7049d7f3ca4293de5bfb972c43750/Patterns/German/German-DateTime.yaml#L279-L281